week-4-practise/level-1/exercise-1.js

```diff
@@ -38,7 +38,34 @@ var writers = [
     alive: true
   }
 ];
-
+writers.forEach(function(param) {
+  console.log(
+    "Hi, my name is " +
+      param.firstName +

```

Comment:
Need a space in here, I think.

Reply:
I have already put a space after my name is  '. is it necessery to put like +  '  ' this?

Reply:
It prints the names joined together, e.g. "VirginiaWolf" instead of "Virginia Wolf". It's just a very small observation.